documentation/asciidoc/topics/proc_connecting_console.adoc

```diff
@@ -0,0 +1,12 @@
+[id='creating_caches_console-{context}']
+= Accessing {brandname} Console
+Access the console to create caches, perform adminstrative operations, and monitor your {brandname} clusters.
+
+.Prerequisites
+
+* Retrieve the service hostname and, if necessary, port where the console is available.
```

Comment:
For the console isn't this only possible if a route/loadbalancer/ingress is used? A user can't access the console with just a service as it's not externally accessible by the user's browser.

Reply:
@ryanemerson I remember accessing the console via load balancer from the OCP cluster that @Crumby has for testing. 

maybe we should remove these instructions for the console altogether?

Reply:
> @ryanemerson I remember accessing the console via load balancer from the OCP cluster that @Crumby has for testing.

That makes sense. My issue was that I thought these instructions were talking about connecting directly via the normal service.

Reply:
I see. Maybe it's somewhat overloaded to say "service hostname" when we mean the load balancer or node port? I suppose Route isn't technically a service either. 

Should we change to maybe "$NETWORK_HOSTNAME" to denote the hostname/url/IP address that provides access to the Infinispan cluster? This could also be the name of the internal service that the operator creates.

Reply:
I'm not sure if a number variable placeholder helps or not tbh. Maybe we need to leave this sentence as is and then add a note somewhere in this section explicilty saying that in order to access the console, you need to configure one of the external services, route etc.